spyder/preferences/shortcuts.py

```diff
@@ -681,14 +688,22 @@ def adjust_cells(self):
 
     def load_shortcuts(self):
         """Load shortcuts and assign to table model."""
+        # item[1] -> context, item[2] -> name
+        shortcut_data = [(item[1], item[2]) for item in self.shortcut_data]

```

Comment:
@ccordoba12 there is no possibility of clash since we only check if the (context, name) of the `iter_shortcuts` appear on the registered actions in the `shortcut_data`